website/docs/reference/warehouse-profiles/snowflake-profile.md

```diff
@@ -66,7 +66,7 @@ my-snowflake-db:
 
 To use SSO authentication for Snowflake, omit a `password` and instead supply an `authenticator` config to your target. `authenticator` can be one of 'externalbrowser' or a valid Okta URL.
 
-**Note**: By default, every connection that dbt opens will require you to re-authenticate in a browser. Contact your Snowflake support rep and inquire about turning on the "id token cache" for your account as described [here](https://github.com/snowflakedb/snowflake-connector-python/issues/140#issuecomment-447028785).
+**Note**: By default, every connection that dbt opens will require you to re-authenticate in a browser. The Snowflake connector package supports cacheing your session token, but it [currently only supports Windows and Mac OS](https://docs.snowflake.com/en/user-guide/odbc-parameters.html#using-browser-based-sso-with-connection-caching-macos-and-windows-only). See [the Snowflake docs](https://docs.snowflake.com/en/user-guide/odbc-parameters.html#using-browser-based-sso-with-connection-caching-macos-and-windows-only) for how to enable this feature in your account.
```

Comment:
We've upgraded the version of snowflake-connector-python and enabled the token cache, but that change is coming in v0.18.0 (currently a release candidate). As such, can you:
* switch the base branch of this PR to the `next` branch
* add `<Changelog>New in v0.18.0</Changelog>` above the section about caching
* link this page in the "New and changed documentation" section of the 0-18-0 migration guide

Reply:
Sorry, I was on holiday over the past 2 weeks so it looks like I missed the boat, nevertheless I've updated the base to `current` as (I think?) you've changed the base branch? 

PS, might be helpful to warn [the others targeting `master` still](https://github.com/fishtown-analytics/docs.getdbt.com/pulls?q=is%3Apr+is%3Aopen+base%3Amaster), it took me a while to work out why I was getting horrific conflicts after I switched to `next` and then tried to switch back to the old base.